dncil/cil/body/__init__.py

```diff
@@ -239,10 +236,9 @@ def parse_basic_blocks(self):
                 # new leader, new basic block
                 bb_curr = BasicBlock(instructions=[insn])
                 self.basic_blocks.append(bb_curr)
-                continue
-
-            assert bb_curr is not None
-            bb_curr.instructions.append(insn)
+            else:
+                assert bb_curr is not None
+                bb_curr.instructions.append(insn)
```

Comment:
Can you provide context for this change? What is it addressing?

Reply:
This change was made in accordance to: https://github.com/mandiant/capa/pull/1326#discussion_r1121640070